scheme/nanopass-scheme/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                nanopass-scheme
+categories          scheme
```

Comment:
I thought about `lang`, but I could not find a similar port there. There's also `devel`. There aren't that many Scheme libraries, but I figured this would act as a starting point. I also noticed the `erlang` category didn't have many libraries. As the main point of this port is ChezScheme, it's not too important to me.